specification/iotoperations/IoTOperations.Management/models/instance.tsp

```diff
@@ -53,32 +53,7 @@ model InstanceProperties {
 
   @doc("The reference to the Schema Registry for this AIO Instance.")
   @pattern("^[a-z0-9][a-z0-9-]*[a-z0-9]$")
-  schemaRegistryNamespace: string;
-
-  @doc("The configuration for components of the AIO Instance.")
-  components?: Components = #{};
-}
-
-@doc("The configuration for components of the AIO Instance.")
-model Components {
-  @doc("The properties of the ADR instance.")
-  adr?: AdrProperties = #{ state: OperationalMode.Enabled };
-
-  @doc("The properties of the AKRI instance.")
-  akri?: AkriInstanceProperties = #{ state: OperationalMode.Enabled };
-
-  @doc("The properties of the connectors instance.")
-  connectors?: ConnectorInstanceProperties = #{
-    state: OperationalMode.Enabled,
-  };
-
-  @doc("The properties of the dataflows instance.")
-  dataflows?: DataflowInstanceProperties = #{ state: OperationalMode.Enabled };
-
-  @doc("The properties of the device provisioning instance.")
-  schemaRegistry?: SchemaRegistryInstanceProperties = #{
-    state: OperationalMode.Enabled,
-  };
+  schemaRegistryId: string;
```

Comment:
it might be good to remove the pattern and use the following TypeSpec capability for this:

https://azure.github.io/typespec-azure/docs/howtos/ARM/resource-type#using-resource-identifiers

Reply:
Ah nice. I wasn't aware of this capability.